content/en/guide/v10/typescript.md

```diff
@@ -92,7 +90,7 @@ function Greeting({ name = "User" }: GreetingProps) {
 }
 ```
 
-Preact also ships a `FunctionComponent` type to annotate anonymous functions. `FunctionComponent` also adds a type for `children`:
+Preact ships a `FunctionComponent` type to annotate anonymous functions. `FunctionComponent` adds a type for `children` to `props`:
```

Comment:
This describes what a type for `children` is added to.
https://github.com/preactjs/preact/blob/f955cfcceb50d504faa7c05a95065512cf900e57/src/index.d.ts#L72-L90